src/Database/Statement/StatementDecorator.php

```diff
@@ -258,9 +257,9 @@ public function fetchAll(string|int $type = self::FETCH_TYPE_NUM): array|false
      * ### Example:
      *
      * ```
-     * $statement = $connection->prepare('SELECT id, title from articles');
+     * $statement = $connection->prepare('UPDATE articles SET foo = 1');
```

Comment:
```suggestion
     * $statement = $connection->prepare('UPDATE articles SET foo = 1 WHERE id = 2');
```
:shrug: might make the example more clear.

Reply:
I would rather the update the description stating it can be used after `UPDATE`, `INSERT` and `DELETE` queries and remove the example altogether. Users would rarely need to call this method directly so a usage e.g. isn't really necessary here.

Reply:
That's fair. Usage can be in the book too.